src/settings/general-settings.tsx

```diff
@@ -222,7 +226,13 @@ const GeneralSettings = (): React.JSX.Element => {
 
 	return (
 		<>
-			<SettingsHeader title={title} onCancel={onCancel} onSave={onSave} isDirty={isDirty} />
+			<SettingsHeader
+				title={title}
+				onCancel={onCancel}
+				onSave={onSave}
+				isDirty={isDirty}
+				hasError={hasError}
```

Comment:
use directly `outOfOfficeError` instead of creating a memo `hasError`
```suggestion
				hasError={outOfOfficeError}
```

Reply:
I think it is better to keep the two logic split. There could be more errors to handle and they can all handled inside that memo